Added in AllowTool and minified everything

ChaosTherum opened the pull request:
So with one fell swoop I minified every single building gone are the day of deconstructing.

Also added in Allowtool

Also normalized hospitality so that it is loaded before MVP mod then patched from MVP

ChaosTherum committed:
Merge remote-tracking branch 'refs/remotes/simon-82/dev' into dev

ChaosTherum committed:
Normalized

Normalized with pack conventions

ChaosTherum committed:
Added allow tool back in

It was pretty much drag and drop

ChaosTherum committed:
Minified literally every building

Basically added the minified option to the building base

ChaosTherum committed:
Cleaned xml

simon-82 commented:
- Hospitality still loads after the modpack folder in the modorder. So now your changes to the bed should get reverted by hospitality itself 
- changing the building base like that seems very risky. Needs testing first
- wasn't there an update to allowtool or some new features? Ah yes https://ludeon.com/forums/index.php?topic=18377.0 
  Maybe we could use that instead?

ChaosTherum committed:
Fixes and changes

Removed allowtool and added haul priority instead. Also changed the load
order so that hospitality loads before mvp

ChaosTherum committed:
Removed minification

ChaosTherum committed:
Fixed haul priority.

simon-82 commented:
great :smiley:
